css/style.css

```diff
@@ -0,0 +1,555 @@
+  font-weight: 700;
+  color: #fff;
+  text-transform: uppercase;
+  max-width: 195px;
+  margin-bottom: 5px;
+}
+
+.button {
+  text-transform: uppercase;
+  font-size: 12px;
+  text-decoration: none;
+  cursor: pointer;
+  font-weight: 700;
+}
+
+.button:hover {
+  text-decoration: underline;
+}
+
+.container__button {
+  color: #fff;
+}
+
+.products {
+  padding-bottom: 45px;
+}
+
+.title {
+  font-size: 22px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.products__title {
+  text-align: center;
+  padding: 32px 0 0 0;
+}
+
+.products__items {
+  display: flex;
+  justify-content: space-around;
+}
+
+.sneakers {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 261px;
+}
+
+.sneakers__image {
+  width: 188px;
+  height: 173px;
+  padding-bottom: 30px;
+}
+
+.sneakers__name {
+  font-size: 12px;
+  margin: 0;
+}
+
+.sneakers__description {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex: 1;
+}
+
+.sneakers__description_price {
+  font-size: 14px;
+  font-weight: 900;
+  margin: 5px 0;
+  flex: 3;
+}
+
+.sneakers__description_colors {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex: 1;
+}
+
+.color_mobile {
+  display: none;
+}
+
+.products__items_border {
+  margin: 43px 30px;
+  height: 100px;
+  max-width: 1px;
+  flex: 1;
+}
+
+.history {
+  background: #000;
+  height: 494px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.history__video {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-left: 32px;
+  padding-right: 45px;
+}
+
+.history__video_main {
+  position: relative;
+  width: 644px;
+  height: 358px;
+}
+
+.history__video_background {
+  position: absolute;
+}
+
+.history__video_play {
+  position: absolute;
+}
+
+.history__text {
+  width: 266px;
+}
+
+.history__title {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0;
+}
+
+.history__text_main {
+  font-size: 14px;
+}
+
+.history__button {
+  color: #fff;
+}
+
+@media screen and (max-width: 1169px) {
+  body {
+    max-width: 930px;
+    margin: 0 auto;
+  }
+
+  .header__menu {
+    height: 130px;
+  }
+
+  .header__container {
+    height: 253px;
+  }
+
+  .history__text_main {
+    font-size: 13px;
+    font-weight: normal;
+  }
+
+  .products__title {
+    padding: 50px 0 0 0;
+  }
+
+  .products {
+    padding-bottom: 60px;
+  }
+
+  .history {
+    height: 442px;
+  }
+
+  .history__video_main {
+    width: 575px;
+    height: 325px;
+  }
+
+  .history__text {
+    width: 248px;
+  }
+}
+
+@media screen and (max-width: 1023px) {
+  body {
+    max-width: 1023px;
+    margin: 0 auto;
+  }
+
+  .header__menu {
+    justify-content: space-between;
+  }
+
+  .header__logo {
+    display: none;
+  }
+
+  .header__logo_mobile {
+    display: flex;
+    margin-left: 43px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .menu__form {
+    display: none;
+  }
+	
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5985384)